packages/forms/src/Components/Wizard.php

```diff
@@ -65,6 +67,18 @@ public function steps(array | Closure $steps): static
         return $this;
     }
 
+    public function startOnStep(int $startOnStep): static
+    {
+        $this->startOnStep = $startOnStep - 1;
```

Comment:
This logic belongs in the view

```suggestion
        $this->startStep = $startStep;
```